astropy/wcs/wcsapi/wrappers/sliced_wcs.py

```diff
@@ -243,6 +243,8 @@ def pixel_to_world_values(self, *pixel_arrays):
         return world_arrays
 
     def world_to_pixel_values(self, *world_arrays):
+        sliced_out_world_coords = self._pixel_to_world_values_all(*[0]*len(self._pixel_keep))
```

Comment:
I have a slight concern that pixel value 0 might not always be inside a region where the WCS returns a real value (and not NaN) - why not use the actual pixel index used for the slicing?

Reply:
This method actually does do that (or it wouldn't work) this input is just specifying the world coordinates for the not sliced axes.